src/sage/graphs/graph_decompositions/slice_decomposition.pyx

```diff
@@ -0,0 +1,970 @@
+            sage: SD.xslice_data('a')['sequence']
+            [['a'], ['b', 'c', 'd', 'e', 'f'], ['g']]
+
+            sage: SD.active_edges('b') == SD.xslice_data('b')['active_edges']
+            True
+
+            sage: SD.lexicographic_label('u')
+            ['a', 'b', 'c', 'd', 'e', 'f', 'g']
+            sage: SD.xslice_data('u')['lexicographic_label']
+            ['a', 'b', 'c', 'd', 'e', 'f', 'g']
+
+        TESTS:
+
+            sage: G = graphs.RandomGNP(15, 0.3)
+            sage: SD = G.slice_decomposition()
+            sage: all(SD[v]['slice'] == SD.slice(v) for v in G)
+            True
+            sage: all(SD[v]['sequence'] == SD.xslice_sequence(v) for v in G)
+            True
+            sage: all(SD[v]['active_edges'] == SD.active_edges(v) for v in G)
+            True
+            sage: all(SD[v]['lexicographic_label'] == SD.lexicographic_label(v) for v in G)
+            True
+
+            sage: SD = graphs.PetersenGraph().slice_decomposition()
+            sage: SD['John']
+            Traceback (most recent call last):
+            ...
+            LookupError: vertex (John) does not appear in the slice decomposition
+        """
+        if v not in self.sigma_inv:
+            raise LookupError(f"vertex ({v}) does not appear in the slice "
+                              "decomposition")
+        i = self.sigma_inv[v]
+        return {'pivot': v,
+                'slice': self._slice(i),
+                'sequence': self._xslice_sequence(i),
+                'lexicographic_label': self._xslice_lex_label(i),
+                'active_edges': self._xslice_active_edges(i),
+                }
+
+    def lexBFS_order(self):
+        r"""
+        Return the lexBFS order corresponding to the slice decomposition.
+
+        EXAMPLES:
+
+        ::
+
+            sage: from sage.graphs.traversals import _is_valid_lex_BFS_order
+            sage: G = graphs.PetersenGraph(); SD = G.slice_decomposition()
+            sage: SD.lexBFS_order()
+            [0, 1, 4, 5, 2, 6, 3, 9, 7, 8]
+            sage: _is_valid_lex_BFS_order(G, SD.lexBFS_order())
+            True
+
+        TESTS:
+
+            sage: from sage.graphs.traversals import _is_valid_lex_BFS_order
+            sage: for _ in range(5):
+            ....:   G = graphs.RandomGNP(15, 0.3)
+            ....:   SD = G.slice_decomposition()
+            ....:   _is_valid_lex_BFS_order(G, SD.lexBFS_order())
+            True
+            True
+            True
+            True
+            True
+
+        """
+        return list(self.sigma)
+
+    def xslice_data(self, v):
+        r"""
+        Return the data about the x-slice of the vertex `v`.
+
+        This method is a wrapper around :meth:`SliceDecomposition.__getitem__`
+
+        TESTS:
+
+            sage: G = graphs.RandomGNP(15, 0.3)
+            sage: SD = G.slice_decomposition()
+            sage: all(SD[v] == SD.xslice_data(v) for v in G)
+            True
+
```

Comment:
remove this empty line

Reply:
done in 8ffab789b46d677789f96faee3917ef439c667a3